src/MessageItem.res

```diff
@@ -26,19 +29,19 @@ let make = (~message: message) => {
     <p> {textEl(message.content)} </p>
     {if Array.length(message.attachments) > 0 {
       <div>
-        <b> {textEl(`Anhänge`)} </b>
+        <b> {"Anhänge"->React.string} </b>
```

Comment:
The convention for handling react elements is to use `->React.string` etc. Might look like its more to type, but generally speaking it spares you lots of headaches when moving code between codebases later on, since its a community convention.

Reply:
Also notice how this is using a `""` string for a string with umlauts.